Doxyfile.in

```diff
@@ -618,7 +618,7 @@ CITE_BIB_FILES         =
 # The QUIET tag can be used to turn on/off the messages that are generated
 # by doxygen. Possible values are YES and NO. If left blank NO is used.
 
-QUIET                  = NO
+QUIET                  = YES
```

Comment:
It just spews a bunch of processing .... / generating .... messages which are useless. I'm all for displaying warnings / errors but this is over the top especially with many files.